include/seastar/json/json_elements.hh

```diff
@@ -297,8 +297,10 @@ struct json_return_type {
    json_return_type(json_return_type&& o) noexcept : _res(std::move(o._res)), _body_writer(std::move(o._body_writer)) {
    }
     json_return_type& operator=(json_return_type&& o) noexcept {
+      if (this != &o) {
```

Comment:
It looks like you are breaking the indentation?

Reply:
I used half-indent to emphasize the net-change.
I can reindent to make the code more aesthetic.

Reply:
Please do. This is a three-line change, I think we can handle it in the review. Or, if you prefer, reindent in yet another patch.
I don't think we should keep wrong indentation in the source for years.